pkg/config/admission/config.go

```diff
@@ -0,0 +1,43 @@
+package admission
+
+import (
+	"strings"
+
+	"github.com/horizoncd/horizon/pkg/admission/models"
+)
+
+type FailurePolicy string
+
+func (f FailurePolicy) Eq(other FailurePolicy) bool {
+	return strings.EqualFold(string(f), string(other))
+}
+
+const (
+	FailurePolicyIgnore FailurePolicy = "ignore"
+	FailurePolicyFail   FailurePolicy = "fail"
+)
+
+type ClientConfig struct {
+	URL      string `yaml:"url"`
+	CABundle string `yaml:"caBundle"`
+	Insecure bool   `yaml:"insecure"`
+}
+
+type Rule struct {
+	Resources  []string           `yaml:"resources"`
+	Operations []models.Operation `yaml:"operations"`
+	Versions   []string           `yaml:"versions"`
+}
+
+type Webhook struct {
+	Name           string        `yaml:"name"`
+	Kind           models.Kind   `yaml:"kind"`
+	FailurePolicy  FailurePolicy `yaml:"failurePolicy"`
+	TimeoutSeconds int32         `yaml:"timeoutSeconds"`
```

Comment:
Can we use "time.Duration" as the type of "Timeout" field? Then we can simply use "30s" as the value of the field in the config file, which will be more meaningful.

Reply:
fixed